--- sdcv-0.4.2-orig/src/Makefile.am	2006-04-24 02:49:40.000000000 -0500
+++ sdcv-0.4.2/src/Makefile.am	2006-05-09 18:07:26.734375000 -0500
@@ -7,8 +7,8 @@
 	utils.cpp utils.hpp
 
 sdcv_DEPENDENCIES = lib/libstardict.a
-sdcv_LDADD = @SDCV_LIBS@ @LIBINTL@ @LIBREADLINE@ lib/libstardict.a
+sdcv_LDADD = lib/libstardict.a @SDCV_LIBS@ @LIBINTL@ @LIBREADLINE@
 localedir = $(datadir)/locale
 
-INCLUDES = @SDCV_CFLAGS@  -I$(top_builddir) -Ilib
+INCLUDES = @SDCV_CFLAGS@  -I$(top_builddir) -I$(srcdir)/lib
 DEFS=-DLOCALEDIR=\"$(localedir)\" @DEFS@
